client/src/ui/components/quest/steps/createAttackArmy.tsx

```
@@ -0,0 +1,125 @@
+    },
+    classes: "-mt-5",
+    beforeShowPromise: function () {
+      return waitForElement(".attack-army-selector");
+    },
+    advanceOn: {
+      selector: ".attack-army-selector",
+      event: "click",
+    },
+    showOn: () => {
+      const showStep = document.querySelector(".attacking-army-selector");
+      return !Boolean(showStep);
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)